quotes/out/cldr.inc

```diff
@@ -128,6 +128,7 @@
 :root:lang(ru),       :not(:lang(ru)) > :lang(ru)             { quotes: '\00ab' '\00bb' '\201e' '\201c' } /* &#x00ab; &#x00bb; &#x201e; &#x201c; */
 :root:lang(rw),       :not(:lang(rw)) > :lang(rw)             { quotes: '\00ab' '\00bb' '\2018' '\2019' } /* &#x00ab; &#x00bb; &#x2018; &#x2019; */
 :root:lang(rwk),      :not(:lang(rwk)) > :lang(rwk)           { quotes: '\201c' '\201d' '\2018' '\2019' } /* &#x201c; &#x201d; &#x2018; &#x2019; */
+:root:lang(sah),      :not(:lang(sah)) > :lang(sah)           { quotes: '\00ab' '\00bb' '\201e' '\201c' } /* &#x00ab; &#x00bb; &#x201e; &#x201c; */
```

Comment:
I see `sah` is Sakha/Yakut, another Russian-affiliated Cyrillic lang, and as with the `be` change this matches what we have for `ru`